tools/GenerateTestSummary/Program.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.CommandLine;
+using Aspire.TestTools;
+
+// Usage: dotnet tools run GenerateTestSummary --dirPathOrTrxFilePath <path> [--output <output>] [--combined]
+// Generate a summary report from trx files.
+// And write to $GITHUB_STEP_SUMMARY if running in GitHub Actions.
+
+var dirPathOrTrxFilePathArgument = new Argument<string>("dirPathOrTrxFilePath");
```

Comment:
I'm struggling to understand all use-cases for this argument; more comments would be really helpful.

I understand when this points to a folder where *.trx files are, but what does it mean when it is a file? Why would I pass a file?

What does the following combination means?
```
 -- D:\a\aspire\aspire/testresults -u https://github.com/dotnet/aspire/actions/runs/14966104757/artifacts/3103892024
```